src/items.h

```diff
@@ -208,6 +208,7 @@ class ItemType
 		int32_t extraDefense = 0;
 		int32_t armor = 0;
 		uint16_t rotateTo = 0;
+		uint16_t unmoveabledItem = 0;
```

Comment:
Shouldn't that be a boolean? What the hell "unmoveabled" means and is it telling you that it can be saved in a house or will it confuse ppl with the difference between "moveable" member?

Reply:
I wasn't using as a boolean and that worked as well too, but I'll move it to a boolean.

Reply:
Same as @DanielChabrowski 
IMO it should be boolean.
Also I would change the name to isCarpet.